applications/machine_learning/configuration/nrf54h20dk_nrf54h20_cpuapp/sensor_manager_def.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <caf/sensor_manager.h>
+
+/* This configuration file is included only once from sensor_manager module and holds
+ * information about the sampled sensors.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} sensor_manager_def_include_once;
+
+static struct sm_trigger sensor_trigger = {
+	.cfg = {
+		.type = SENSOR_TRIG_MOTION,
+		.chan = SENSOR_CHAN_ACCEL_XYZ,
+	},
+	.activation = {
+		.type = ACT_TYPE_ABS,
+		.thresh = FLOAT_TO_SENSOR_VALUE(CONFIG_ADXL362_ACTIVITY_THRESHOLD *
+			(IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_8G) ? 8.0 :
+			 IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_4G) ? 4.0 :
+			 IS_ENABLED(CONFIG_ADXL362_ACCEL_RANGE_2G) ? 2.0 : 1/0)
```

Comment:
Is the division here correct?

Reply:
It looks like it is not correct. Corrected.